src/language-css/print/comma-separated-value-group.js

```diff
@@ -269,7 +269,8 @@ function printCommaSeparatedValueGroup(path, options, print) {
       options.parser === "scss" &&
       isMathOperator &&
       iNode.value === "-" &&
-      iNextNode.type === "value-func"
+      iNextNode.type === "value-func" &&
+      locEnd(iNode) !== locStart(iNextNode)
```

Comment:
Is this `-foo()` syntax only supported by SCSS? I don't think LESS support it. So, we should check `options.parser` here.

Reply:
Thanks, updated it